picoquictest/h3zerotest.c

```diff
@@ -74,7 +74,7 @@ static h3zero_test_integer_case_t h3zero_int_case[] = {
 
```

Comment:
It seems that the only changes in these file are formatting changes. If that's the case, maybe we should leave it untouched. But then, we may indeed need a change somewhere, for example to set the default crypto.

Reply:
still unsure what triggered it, will remove it

Reply:
should be moved now to #1061

Reply:
I proposed the usage of clang-format and editorconfig #1062, so future issues like those can be handled fluently during upload. Await your feedback there